compile/compile.go

```diff
@@ -183,8 +184,8 @@ func (c *Compiler) WithBundleVerificationKeyID(keyID string) *Compiler {
 }
 
 // WithCapabilities sets the capabilities to use while checking policies.
-func (c *Compiler) WithCapabilities(capabilities *ast.Capabilities) *Compiler {
-	c.capabilities = capabilities
+func (c *Compiler) WithCapabilities(caps *capabilities.Capabilities) *Compiler {
```

Comment:
All of these break API compat, too, btw. I suppose we don't care too much, do we? 💭